packages/emitter-framework/src/csharp/components/enum-declaration.tsx

```diff
@@ -30,19 +31,26 @@ export function EnumDeclaration(props: EnumDeclarationProps): ay.Children {
   const members = Array.from(type.members.entries());
 
   return (
-    <cs.EnumDeclaration name={name} refkey={refkeys} {...props}>
-      <ay.For each={members} joiner={",\n"}>
-        {([key, value]) => {
-          return (
-            <cs.EnumMember
-              name={cs.useCSharpNamePolicy().getName(key, "enum-member")}
-              refkey={
-                $.union.is(props.type) ? efRefkey(props.type.variants.get(key)) : efRefkey(value)
-              }
-            />
-          );
-        }}
-      </ay.For>
-    </cs.EnumDeclaration>
+    <>
+      <cs.EnumDeclaration name={name} refkey={refkeys} {...props}>
+        <ay.For each={members} joiner={",\n"}>
+          {([key, value]) => {
+            return (
+              <>
+                <cs.DocWhen doc={getDocComments($, value)} />
+                <cs.EnumMember
+                  name={cs.useCSharpNamePolicy().getName(key, "enum-member")}
```

Comment:
do you actually need to call the policy here? (I know it was there before but maybe there is some other cases where this could be simplified)